src/pages/postgraphile/smart-tags.md

```diff
@@ -325,9 +325,16 @@ Since version
 
 Applies to:
 
-- Functions returning `SETOF`: adds the `orderBy` argument to this field
-- Computed column functions with no required parameters: allows this function to
-  be used as an order in the `orderBy` argument for the parent table
+- Functions returning `SETOF`: adds the `orderBy` argument to this connection,
+  allowing to _sort_ the set by any of its scalar fields.
+- Computed column functions with no required parameters and returning a scalar
+  or array: allows this function to be used as an order in the `orderBy`
+  argument for the parent table, allowing to _sort by_ this function's value the
+  parent table.
+
+If used on a computed function that returns a SETOF _record type_ it will make
+the connection _sortable_ (like it would with any other `SETOF` function) and
+NOT a _sort by_ for its parent table
```

Comment:
With the edits above, I don't think this paragraph is necessary? It might be helpful to say that `@sortable has no effect on functions that return SETOF scalar` or similar?

Reply:
I agree, not necessary anymore. I'll add a warning on scalar `SETOF`, better to cover as much as possible to minimise questions